demos/social_distance_demo/social_distance_demo.hpp

```diff
@@ -0,0 +1,117 @@
+DEFINE_string(m, "", detection_model_message);
+DEFINE_string(m_reid, "", reid_model_message);
+DEFINE_string(d, "CPU", target_device_message);
+DEFINE_string(d_reid, "CPU", target_device_message_reid);
+DEFINE_bool(pc, false, performance_counter_message);
+DEFINE_bool(r, false, raw_output_message);
+DEFINE_double(t, 0.85, thresh_output_message);
+DEFINE_string(c, "", custom_cldnn_message);
+DEFINE_string(l, "", custom_cpu_library_message);
+DEFINE_bool(no_show, false, no_show_processed_video);
+DEFINE_bool(auto_resize, false, input_resizable_message);
+DEFINE_uint32(nireq, 0, ninfer_request_message);
+DEFINE_uint32(nc, 0, num_cameras);
+DEFINE_string(fpga_device_ids, "", fpga_device_ids_message);
+DEFINE_bool(loop_video, false, loop_video_output_message);
+DEFINE_uint32(n_iqs, 3, input_queue_size);
+DEFINE_uint32(ni, 0, ninputs_message);
+DEFINE_uint32(fps, 0, fps);
+DEFINE_uint32(n_wt, 1, worker_threads);
+DEFINE_string(display_resolution, "1920x1080", display_resolution_message);
+DEFINE_bool(tag, false, use_tag_scheduler_message);
+DEFINE_uint32(nthreads, 0, infer_num_threads_message);
+DEFINE_string(nstreams, "", infer_num_streams_message);
+DEFINE_string(u, "", utilization_monitors_message);
+
+/**
+* \brief This function show a help message
+*/
+void showUsage() {
+    std::cout << std::endl;
+    std::cout << "interactive_vehicle_detection [OPTION]" << std::endl;
+    std::cout << "Options:" << std::endl;
+    std::cout << std::endl;
+    std::cout << "    -h                         " << help_message << std::endl;
+    std::cout << "    -i \"<path1>\" \"<path2>\"     " << video_message << std::endl;
+    std::cout << "    -m \"<path>\"                " << detection_model_message << std::endl;
+    std::cout << "    -m_reid \"<path>\"           " << reid_model_message << std::endl;
+    std::cout << "      -l \"<absolute_path>\"     " << custom_cpu_library_message << std::endl;
+    std::cout << "          Or" << std::endl;
+    std::cout << "      -c \"<absolute_path>\"     " << custom_cldnn_message << std::endl;
+    std::cout << "    -d \"<device>\"              " << target_device_message << std::endl;
+    std::cout << "    -d_reid \"<device>\"         " << target_device_message_reid << std::endl;
+    std::cout << "    -pc                        " << performance_counter_message << std::endl;
+    std::cout << "    -r                         " << raw_output_message << std::endl;
+    std::cout << "    -t                         " << thresh_output_message << std::endl;
+    std::cout << "    -no_show                   " << no_show_processed_video << std::endl;
+    std::cout << "    -auto_resize               " << input_resizable_message << std::endl;
+    std::cout << "    -nireq                     " << ninfer_request_message << std::endl;
+    std::cout << "    -nc                        " << num_cameras << std::endl;
+    std::cout << "    -fpga_device_ids           " << fpga_device_ids_message << std::endl;
```

Comment:
FPGA? Have you tested this? In general, unless we really test the demo on a particular device, we should not mention it as supported.